server/src/internalClusterTest/java/org/opensearch/action/bulk/BulkWithUpdatesIT.java

```diff
@@ -79,6 +80,7 @@
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.nullValue;
 
+@LuceneTestCase.AwaitsFix(bugUrl = "Needs more deep dive")
```

Comment:
May be let these tests run ? We will get more data around failure

Reply:
Sure, will remove this